src/pages/administration/userRole/users.jsx

```diff
@@ -39,50 +41,66 @@ const Users = () => {
           })
         );
       });
-  }
+  };
 
   const columns = useMemo(
     () => [
       {
-        Header: 'Name',
-        accessor: 'userName',
-      },
-      {
-        Header: 'Email',
-        accessor: 'uid',
+        Header: "Name",
+        accessor: "userName",
       },
       {
-        Header: 'Role',
-        accessor: 'role',
-        Cell: ({ cell: { value } }) => value || "-"
+        Header: "Email",
+        accessor: "uid",
       },
       {
-        Header: 'Add Role to Users',
-        accessor: 'Editing',
+        Header: "Users Role",
+        accessor: "Editing",
         Cell: (cellObj) => (
-          <select required onChange={(change) => handleClickEditRow(cellObj, change)} className={classes.select}>
+          <select
+            required
+            onChange={(change) => handleClickEditRow(cellObj, change)}
+            className={classes.select}
+          >
             <option></option>
             <option value="Students">Students</option>
             <option value="Teachers">Teachers</option>
             <option value="Administration">Administration</option>
           </select>
-        )
+        ),
       },
+      // {
+      //   Header: "Role",
+      //   accessor: "role",
+      //   Cell: ({ cell: { value } }) => value || "-",
+      // },
     ],
     []
-  )
+  );
+
+  const saveButtonHanler = () => {};
```

Comment:
What is that? why is empty

Reply:
I need it next time when the administrator needs to save changes